### ISSUE TITLE
元モデル

### ISSUE THREAD

**Issue by taiking0905** [taiking0905/LearningColorConstancy#11]
Step2 学習状況まとめ（簡易版）

Optimizer: Adam

Learning Rate: 3e-4 / 30 ≈ 1e-5

Dropout: 0.5

ERASE Size: 60

Epoch: 200〜1000

学習状況

train loss ≈ 0 → 訓練データにほぼフィット

val loss ≈ 0.0014 → 少し汎化誤差あり

Step2での改善はほとんど進んでいない状態

改善の余地

Learning Rate を少し上げる（例: 3e-5）

正則化やデータ拡張を少し強める

深い層だけ微調整する

結論

現状でもほぼベストモデル

少し設定を変えれば val loss をわずかに下げられる余地あり
